llvm/docs/CMake.rst

```diff
@@ -475,6 +475,17 @@ enabled sub-projects. Nearly all of these variable names begin with
 **LLVM_ENABLE_BINDINGS**:BOOL
   If disabled, do not try to build the OCaml bindings.
 
+**LLVM_ENABLE_DEBUGLOC_COVERAGE_TRACKING**:STRING
```

Comment:
Should the ABI-breaking-ness be mentioned here as well...? Not sure if that's a common thing to document here or not.